README.md

```diff
@@ -37,6 +38,29 @@ It also provides the `crud-storage` and `crud-router` roles for
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+## Quickstart
+
+First, [install Tarantool](https://www.tarantool.io/en/download).
+
+Now you have the following options how to learn crud API and use it in a
+project:
+
+* Play with crud on a testing dataset on a single instance:
+
+  ```shell
+  $ git clone https://github.com/tarantool/crud.git
+  $ cd crud
+  $ tarantoolctl rocks install vshard
+  $ ./doc/playground.lua
+  tarantool> crud.select('customers', {{'<=', 'age', 35}})
+  tarantool> crud.select('developers', nil, {first = 6})
+  ```
+* Add crud into dependencies of a Cartridge application and add crud roles into
+  dependencies of your roles (see [Cartridge roles](#cartridge-roles) section).
+* Add crud into dependencies of your application (rockspec, RPM spec -- depends
+  of your choice) and call crud initialization code from storage and router
+  code (see [API](#api) section).
+
```

Comment:
We have a CMakeLists.txt file. Do we need to explain what can be done with it here? Or is it better to do it in a separate issue?

Reply:
In those two bullets: I think, no. A user either add a rock as the dependency (so tarantoolctl rocks will do the job) or add a package as the dependency (so the package will install everything appropriately).

I'll add `tarantoolctl rocks make` to the first bullet.

Reply:
Replaced `tarantoolctl rocks install vshard` with `tarantoolctl rocks make`.